.github/workflows/browser_test.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will do a clean install, start the selenium server, and run
+# all of our browser based test
+
+name: Run browser test nightl
+
```

Comment:
For security porpoises 🐳 can you also add a [permissions property](https://github.com/google/blockly/blob/9442b73d99fb63da4526c651ac41762e458b4ed7/.github/workflows/build.yml#L8)? I think it should just need read and not right.